app/views/field_test/experiments/show.html.erb

```diff
@@ -0,0 +1,117 @@
+<style>
+  details details { padding: 1em 0 0 1.5em; }
+</style>
+<% experiment = @experiment %>
+<h1><%= experiment.name %><% unless experiment.active? %> <small class="closed">Completed</small><% end %></h1>
+<%# NOTE: The root_path is the Rails's engine root path, not the application's root path (e.g. "/") %>
+<p><%= link_to "Back to Experiments", root_path %></p>
+<details<%= " open" if experiment.active? %>>
+  <summary>Experiment Details</summary>
+
+  <% if experiment.description %>
+    <p class="description"><%= experiment.description %></p>
+  <% end %>
+
+  <details<%= " open" if experiment.active? %>>
```

Comment:
Quick, non-blocking suggestion (same as above): let's remove the extra space here if it's not needed.
```suggestion
  <details<%= "open" if experiment.active? %>>
```

Reply:
I need the space because I want the tag to either be `<details>` or `<details open>`; without the space I'll have a tag of `<detailsopen>`